roles/download/defaults/main.yml

```diff
@@ -24,9 +24,9 @@ download_always_pull: False
 download_delegate: "{% if download_localhost %}localhost{% else %}{{groups['kube-master'][0]}}{% endif %}"
 
 # Versions
-kube_version: v1.9.5
+kube_version: v1.10.2
 kubeadm_version: "{{ kube_version }}"
-etcd_version: v3.2.4
+etcd_version: v3.2.16
```

Comment:
you're upgrading etcd too?

Reply:
Matching k8s release notes, not just willy nilly, sir.

Reply:
ok :)